alfresco-transform-misc/alfresco-transform-misc-boot/src/test/resources/quick.alternative.eml

```diff
@@ -1,30 +1,30 @@
-MIME-Version: 1.0

```

Comment:
Format only changes should be done in separate PRs - did anything else change other than line-endings (cc @eknizat @PunchedChimera ) ?

Reply:
There are no changes